cmd/wallet-sdk-gomobile/openid4vp/interaction.go

```diff
@@ -159,6 +160,17 @@ func (o *Interaction) PresentCredential(credentials *verifiable.CredentialsArray
 	return wrapper.ToMobileErrorWithTrace(o.goAPIOpenID4VP.PresentCredential(unwrapVCs(credentials)), o.oTel)
 }
 
+// PresentCredentialUnsafe presents a single credential to redirect uri from
+// request object.
+//
+// Note: this variant of PresentCredential will skip client-side presentation
+// definition constraint validation. All input descriptors will accept the
+// provided credential, at least in terms of issuer fields, and subject data
+// fields.
+func (o *Interaction) PresentCredentialUnsafe(cred *verifiable.Credential) error {
```

Comment:
For consistency with the `PresentCredential` method, this argument should probably be `credential` instead of `cred`. (Sometimes the objective-C bindings require the argument name to be passed in, which means a breaking change if we want to make the naming consistent later)

Reply:
Had to rename it to avoid shadowing the import, would the API look cleaner if I renamed the import instead?

Reply:
From the caller's point of view, it would be slightly cleaner since the argument names are a bit more consistent. But I admit this is a very, very small thing...

Reply:
But I see your point - the import renaming might be a bit ugly too, so... go with your gut :D

Reply:
Did the change, I think you're right :)